docker-compose.yml

```diff
@@ -1,9 +1,39 @@
 version: '3'
+
+networks:
+  lms_default:
+    name: lms_default
+
 services:
   postgres:
     image: postgres:11.5-alpine
+    networks:
+      lms_default: { }
     ports:
       - '127.0.0.1:5433:5432'
     healthcheck:
         test: ["CMD", "pg_isready", "-U", "postgres"]
         interval: 1s
+
+  web:
+    image: hypothesis/lms:dev
+    profiles: ["web"]
```

Comment:
This "profile" should prevent the web service from being started by default unless you specify it by name or select this profile.